.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Thank you for sending a PR !-->
+
+Please perform the following checks and tick mark all the points.
```

Comment:
Suggestion: "tick mark all the points." --> "check all the boxes that apply." (This sounds more idiomatic to me, but I'm open to other opinions.)

By the way, here would be a good place to include the instructions for when the boxes don't apply :)